eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/SiteAccessAware/Contextualizer.php

```diff
@@ -73,6 +78,62 @@ public function mapConfigArray($id, array $config, $options = 0)
             []
         );
 
+        // (!) Keep siteaccess group settings
+        foreach (array_keys($this->availableSiteAccessGroups) as $scope) {
+            $scopeSettings = $config[$this->siteAccessNodeName][$scope][$id] ?? [];
```

Comment:
please extract both blocks inside foreach to separate methods (if possible find common parts), so you have here 
`$this->container->setParameter("$this->namespace.$scope.$id", ...);` calls only